cmd/juju/action/status.go

```diff
@@ -12,6 +12,7 @@ import (
 	"github.com/juju/juju/apiserver/params"
 	"github.com/juju/juju/cmd/modelcmd"
 	"github.com/juju/juju/cmd/output"
+	"time"
```

Comment:
This import belongs in the 'stdlib' group, not at the end of 'github.com/juju/juju/' group.